src/main/utility/utility.c

```diff
@@ -356,3 +356,16 @@ struct timespec utility_timespecFromMillis(int64_t millis) {
         .tv_nsec = (millis % 1000) * 1000000, // ms to ns
     };
 }
+
+
+GString* utility_getFileContent(FILE *stream) {
+    char buf[BUFSIZ];
```

Comment:
I don't think we want to have a hard-coded buffer size. If we can't find some existing utility function that will do the buffer allocation for us, I think I'd lean towards just passing "/dev/stdin" to `utility_getFileContents` for now.

Reply:
Agreed with using "/dev/stdin". (The portability question was [asked on stackoverflow](https://unix.stackexchange.com/questions/36403/how-portable-are-dev-stdin-dev-stdout-and-dev-stderr).)

Reply:
Waoo I find totally amazing that we can open "/dev/stdin" :0
Really good advice thank you !